packages/account/src/Styles/account.scss

```diff
@@ -1104,7 +1104,7 @@ $MIN_HEIGHT_FLOATING: calc(
                 margin-top: 0.5em;
 
                 .account-form__fieldset {
-                    margin-bottom: 3em;
+                    margin-bottom: 4em;
```

Comment:
(throughout) Can we use `rem` unit for this? Does it rely on font-size?

Reply:
We can but I think in this case `em` is better because the required error message gap is dependent on the font size of the error message.

Reply:
Alright, would you please add a comment to it, so it doesn't get automatically replaced in the future by mistake?

Reply:
done